main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.initial
```

Comment:
- [ ] According to the project requirements you need to create a simple console app in the `main.rb` file that works in the same was as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided in the requirements. Main.rb should be the entry point (hint in this [lesson ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point)). You can do that by simply calling the `app.display_option` method in the `main.rb` file. Example snippet:

```rb
equire_relative 'app'

def main
  app = App.new;
  app.display_options
end

main

```